File: app/src/main/java/com/example/myapplication/Slot15/MainActivity.java
package com.example.myapplication.slot15;

import android.os.Bundle;
import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import com.example.myapplication.R;
import com.google.android.material.bottomnavigation.BottomNavigationView;

public class MainActivity extends AppCompatActivity {

    BottomNavigationView bottomNav;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        bottomNav = findViewById(R.id.bottom_nav);
        loadFragment(new FirstFragment());

        bottomNav.setOnItemSelectedListener(item -> {
            Fragment selected = null;
            if (item.getItemId() == R.id.nav_first) selected = new FirstFragment();
            else if (item.getItemId() == R.id.nav_second) selected = new SecondFragment();

            return loadFragment(selected);
        });
    }

    private boolean loadFragment(Fragment fragment) {
        if (fragment == null) return false;
        getSupportFragmentManager().beginTransaction()
                .replace(R.id.fragment_container, fragment)
                .commit();
        return true;
    }
}

File: ProjectPRM/app/src/main/java/com/example/shoestore/activity/MainActivity.kt
//package com.example.shoestore.activity
//
//import android.content.Intent
//import android.os.Bundle
//import android.widget.Toast
//import androidx.appcompat.app.AppCompatActivity
//import androidx.fragment.app.Fragment
//import com.example.shoestore.R
//import com.example.shoestore.databinding.MainActivityBinding
//import com.example.shoestore.fragment.CartFragment
//import com.example.shoestore.fragment.HomeFragment
//import com.example.shoestore.fragment.ProfileFragment
//import com.google.firebase.auth.FirebaseAuth
//
//class MainActivity : AppCompatActivity() {
//    private lateinit var binding: MainActivityBinding
//    private lateinit var authen: FirebaseAuth
//
//    override fun onCreate(savedInstanceState: Bundle?) {
//        super.onCreate(savedInstanceState)
//
//        binding = MainActivityBinding.inflate(layoutInflater)
//        setContentView(binding.root)
//
//        authen = FirebaseAuth.getInstance()
//
//        // Mặc định hiển thị Home
//        loadFragment(HomeFragment())
//
//        // Bắt sự kiện click BottomNavigation
//        binding.bottomNav.setOnItemSelectedListener {
//            when (it.itemId) {
//                R.id.home -> loadFragment(HomeFragment())
//                R.id.profile -> loadFragment(ProfileFragment())
//                R.id.cart -> loadFragment(CartFragment())
//                else -> false
//            }
//            true
//        }
//
//    }
//
//    private fun loadFragment(fragment: Fragment) {
//        supportFragmentManager.beginTransaction()
//            .replace(R.id.fragment_container, fragment)
//            .commit()
//    }
//
//}
package com.example.shoestore.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.example.shoestore.R
import com.example.shoestore.databinding.MainActivityBinding
import com.example.shoestore.fragment.CartFragment
import com.example.shoestore.fragment.HomeFragment
import com.example.shoestore.fragment.ProfileFragment
import com.google.firebase.auth.FirebaseAuth

class MainActivity : AppCompatActivity() {
    private lateinit var binding: MainActivityBinding
    private lateinit var authen: FirebaseAuth

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = MainActivityBinding.inflate(layoutInflater)
        setContentView(binding.root)

        authen = FirebaseAuth.getInstance()

        // Mặc định hiển thị Home
        loadFragment(HomeFragment())

        // Bắt sự kiện click BottomNavigation
        binding.bottomNav.setOnItemSelectedListener {
            when (it.itemId) {
                R.id.home -> loadFragment(HomeFragment())
                R.id.profile -> loadFragment(ProfileFragment())
                R.id.cart -> loadFragment(CartFragment())
                else -> false
            }
            true
        }


    }

    private fun loadFragment(fragment: Fragment) {
        supportFragmentManager.beginTransaction()
            .replace(R.id.fragment_container, fragment)
            .commit()
    }


}